src/openam/values.yaml

```diff
@@ -0,0 +1,95 @@
+# Default values for openam.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: wdijkerman/openam
```

Comment:
Try to use an official repository image if one exists.  There's one with 100k pulls in dockerhub, although it may or may not be official.

Reply:
This is an official repository image (https://hub.docker.com/r/wdijkerman/openam/) which matches my requirement closely. But it still has some limitation. To overcome that, i was planning to write my own image extending this official image in next iteration. Would that be advisable?